mailbox/plugin/quota-search-elasticsearch-v7/src/test/java/org/apache/james/quota/search/elasticsearch/v7/ElasticSearchQuotaSearcherTest.java

```diff
@@ -26,15 +26,17 @@
 
 import org.apache.james.core.Username;
 import org.apache.james.core.quota.QuotaSizeLimit;
-import org.apache.james.quota.search.Limit;
-import org.apache.james.quota.search.Offset;
-import org.apache.james.quota.search.QuotaQuery;
-import org.apache.james.quota.search.QuotaSearchTestSystem;
-import org.apache.james.quota.search.QuotaSearcherContract;
+import org.apache.james.domainlist.api.DomainListException;
+import org.apache.james.mailbox.MessageManager;
+import org.apache.james.mailbox.exception.MailboxException;
+import org.apache.james.quota.search.*;
```

Comment:
We avoid wildcard imports. Maybe you should check and adjust your IDE import settings.

(checkstyle sadly do not run on test classes.)

Reply:
https://github.com/apache/james-project/pull/288/commits/a65445a5a67bd41124021b207ef4ae8d2067540b, seems there were no more classes with this issu